VagrantBuild.sh

```diff
@@ -11,11 +11,12 @@ else # Centos
   source ~/.bash_profile
 fi
 
-HOOT_HOME=~/hoot
+if [ -z "$HOOT_HOME" ]; then
+    HOOT_HOME=~/hoot
+fi
```

Comment:
Added this to all of the provisioning scripts so that you can set `$HOOT_HOME` to wherever you'd like before hand for provisioning a dev VM, or the like, and still have a vagrant VM function properly.